File: jp-livedata-java/src/main/java/com/mao/jp_livedata_java/core/BusAlwaysActiveObserver.java
package com.mao.jp_livedata_java.core;

import androidx.lifecycle.BusLiveData;
import androidx.lifecycle.Observer;

/**
 * @author zhangkun
 * @time 2022/2/16 17:55
 * @Description
 */
public class BusAlwaysActiveObserver<T> extends BaseBusObserverWrapper<T> {

    public BusAlwaysActiveObserver(Observer<T> mObserver, BusLiveData<T> mBusLiveData) {
        super(mObserver, mBusLiveData);
    }
}

File: jp-livedata/src/main/java/com/mao/jp_livedata/core/BusAlwaysActiveObserver.kt
package com.mao.jp_livedata.core

import androidx.lifecycle.BusLiveData
import androidx.lifecycle.Observer

internal class BusAlwaysActiveObserver<T>(private val mObserver: Observer<in T>, private val mBusLiveData: BusLiveData<T>)
    : BaseBusObserverWrapper<T>(mObserver, mBusLiveData)